apidoc/Titanium/UI/iOS/ApplicationShortcuts.yml

```diff
@@ -30,112 +31,169 @@ methods:
   - name: dynamicShortcutExists
     summary: Returns true or false depending if the provided shortcut dictionary already exists.
     parameters:
-      - name: params
-        summary: |
-            The parameters used when creating the dynamic shortcut.  Must include a `type` property, 
-            to determine if the shortcut exists.
-        type: Dictionary
+      - name: itemtype
+        summary: Checks if the dynamic application shortcut item identified by the `itemtype` exists.
+        type: String
     returns:
         type: Boolean        
 
-  - name: addShortcutItem
-    summary: Creates a dynamic application shortcut.
+  - name: addDynamicShortcut
+    summary: Creates a new dynamic application shortcut item.
     parameters:
       - name: params
+        summary: The parameters used when creating a dynamic shortcut.
+        type: ShortcutParams
+
+  - name: removeDynamicShortcut
+    summary: Removes the dynamic application shortcut item identified by the `itemtype`.
+    parameters:
+      - name: itemtype
         summary: |
-            The parameters used when creating the dynamic shortcut.  Must include the `type` and `title` properties.
-            Properties are:
-            	* type - the unique key for the application shortcut
-            	* title - the title of the application shortcut
-            	* subtitle - the subtitle displayed on the application shortcut
-            	* icon - the icon to be displayed on the application shortcut, for example Titanium.UI.iOS.SHORTCUT_ICON_TYPE_COMPOSE
-        type: Dictionary
-
-  - name: removeShortcutItem
-    summary: Removes the provided application shortcut item.
+            Use the `itemtype` property to determine which shortcut should be removed.
+        type: String
+
+  - name: getDynamicShortcut
+    summary: Gets the dynamic application shortcut item identified by the `itemtype`.
```

Comment:
Should add that it is "read-only" - that changing the returned object will not have any effect.